packages/namekit-react/src/components/Identity.tsx

```diff
@@ -12,21 +12,46 @@ import {
 } from "@namehash/nameguard";
 import cc from "classcat";
 
-export interface ProfileLinkConfig {
-  getProfileURL: (address: string) => string;
-  getProfileLink: (address: string, children: ReactNode) => JSX.Element;
+export class ProfileLinkGenerator {
+  private baseURL: string;
+  private name: string;
+
+  constructor(name: string, baseURL: string) {
+    this.name = name;
+    this.baseURL = baseURL;
+  }
+
+  getName(): string {
```

Comment:
Can you help me understand the goal for `getName`? It will help a lot to document ideas in comments.